MCprep_addon/world_tools.py

```diff
@@ -1206,6 +1206,19 @@ def execute(self, context):
 # Below for register
 # -----------------------------------------------------------------------------
 
+def mcprep_sun_ease(world_time):
+    x = world_time % 24
+	f_x = (-0.073)*(x**2)+(1.762)*(x)-(5.161)
+	
+	# We want to only return this value if it's above 2, otherwise ignore it
+	return f_x if f_x >= 2 else 0
+
+def mcprep_moon_ease(world_time):
+    x = world_time % 24
+    if x >= 18 and x <= 23.999:
+        return (-0.014)*(x**2)+(0.831)*(x)-(10.166)
+    else:
+        return (-0.012)*(x**2)-(0.164)*(x)+(1.590)
```

Comment:
So I'm realizing now, one issue with this approach of defining drivers, is that it will not stand on its own. ie if sent to another computer to render, if it doesn't have MCprep installed, it will fail to work.

Something for us to think about further. We could potentially have this as a script file which is actually generated into a text block that lives with the blend file for instance, but I do want to have a chance to play around with this more first before you go down this path.

Reply:
> Something for us to think about further. We could potentially have this as a script file which is actually generated into a text block that lives with the blend file for instance

We could also append the text file from the file since it's included